Archipelago.MultiClient.Net.Analyzers/Archipelago.MultiClient.Net.Analyzers.Test/ItemFlagsTest.cs

```diff
@@ -1,267 +0,0 @@
-﻿using Microsoft.CodeAnalysis.Testing;
```

Comment:
This file (testing diagnostics and fixes for == comparisons) should be kept, I assume you meant to rename it (my preference) or merge it with the one for switches